src/DateTimeDefault.ts

```diff
@@ -107,7 +110,10 @@ export function defaultParseToTime(timeItems: string[], timeFormat: string): May
   return {hours, minutes, seconds}
 }
 
-export function defaultParseToDate(dateItems: string[], dateFormat: string): Maybe<SimpleDate> {
+function defaultParseToDate(dateItems: string[], dateFormat: Maybe<string>): Maybe<SimpleDate> {
+  if(dateFormat === undefined) {
+    return undefined
+  }
```

Comment:
Codecov says this condition is not covered by tests. Please add one

Formatting:
```suggestion
  if (dateFormat === undefined) {
    return undefined
  }
```